### ISSUE TITLE
QT5 LNet installation error: Forward declaration not solved 

### ISSUE THREAD

**Issue by alanphys** [almindor/lnet#7]
**Problem**
Trying to install lnet in Lazarus from the online package manager and from git gives 

lclnet.pas(45,15) Error: Forward declaration not solved "HandleIgnoreError(TLHandle);"
lclnet.pas(46,15) Error: Forward declaration not solved "HandleIgnoreWrite(TLHandle);"
lclnet.pas(47,15) Error: Forward declaration not solved "HandleIgnoreRead(TLHandle);"
lclnet.pas(48,15) Error: Forward declaration not solved "InternalUnplugHandle(TLHandle);"
lclnet.pas(50,15) Error: Forward declaration not solved "HandleEvents(Int64;LongWord);"
lclnet.pas(58,14) Error: Forward declaration not solved "AddHandle(TLHandle):Boolean;"

**System**
Fedora 30
KDE 5/QT 5.12.4
Lazarus 2.0.6
LNet 0.66

**Cause**
QT5 not defined in lclnet.pas

**Workaround**
Add
`{$ifdef LCLQT5}
  {$i lclgtkeventer.inc} // identical code ;)
{$endif}`
before end statement in lclnet.pas

Regards
Alan

**Comment by almindor** [comment]
Hey, I think the preprocessor ifdefs changed in newer lazarus and need to
be updated. You need to use the QT "lcl" eventer for this tho otherwise it
won't work as the main "event loop" won't connect.

I find it interesting that the gtk one worked for you.

Ales

st 5. 2. 2020 o 3:03 Alan Chamberlain <notifications@github.com> napísal(a):

**Comment by alanphys** [comment]
Yes, the new version defines LCLQT5. I'm not seeing an lclqteventer.inc. I just copied your existing LCLQT declaration which uses the lclgtkeventer.inc.

So far everything seems to be running OK, but I am only using the TLHTTPServer component.
Regards
Alan
